hydromt_wflow/wflow.py

```diff
@@ -900,6 +901,8 @@ def setup_lulcmaps(
         * **PathFrac** map: The fraction of compacted or urban area per grid cell [-]
         * **WaterFrac** map: The fraction of open water per grid cell [-]
         * **N** map: Manning Roughness [-]
+        * **alpha_h1** map: Root water uptake reduction at soil water pressure head h1
+          (0.0 or 1.0) [-]
```

Comment:
If should be integer can you update here?
```suggestion
          (0 or 1) [-]
```